report_xml/__manifest__.py

```diff
@@ -2,15 +2,33 @@
 # License AGPL-3.0 or later (https://www.gnuorg/licenses/agpl.html).
 {
     "name": "XML Reports",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Reporting",
     "website": "https://github.com/OCA/reporting-engine",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": (
+        "Tecnativa"
+        ", Odoo Community Association (OCA)"
+        ", Avoin.Systems"
+    ),
     "license": "AGPL-3",
     "installable": True,
     "application": False,
     "summary": "Allow to generate XML reports",
     "depends": ["web"],
-    "data": ["views/webclient_templates.xml", "views/ir_actions_views.xml"],
-    "demo": ["demo/report.xml"],
+    "data": [
+        "views/webclient_templates.xml",
+        "views/ir_actions_report_view.xml",
+    ],
+    "demo": [
+        "demo/report.xml",
+        "demo/demo_report.xml",
+    ],
+    # `post_load`: manipulations before any model or data has been initialized.
+    # "post_load": "post_load",
+    # `pre_init_hook`: manipulations before installing the module.
+    # "pre_init_hook": "pre_init_hook",
+    # `post_init_hook`: manipulations after installing the module.
+    "post_init_hook": "post_init_hook",
```

Comment:
Why do you need a post_init_hook?

Reply:
To add XSD Validation Schema for a demo report if it's in the system. Demo data records are always created with `noupdate == True` and render of [tag `report`](https://github.com/odoo/odoo/blob/13.0/odoo/tools/convert.py#L276) doesn't support new `ir.actions.report` field `xsd_schema`. Thus it is impossible to define `xsd_schema` in the demo definition or add schema after that via xml update record. Therefore it possible to add value to `xsd_schema` field for demo record only via hook.

This problem also mentioned [here](https://github.com/Tatider/reporting-engine/blob/13.0-mig-report_xml/report_xml/demo/report.xml#L13)

Reply:
Ok, clean the unnecessary comments :wink:

Reply:
done